pkg/remoting/getty/getty_client_test.go

```diff
@@ -0,0 +1,163 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package getty
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/agiledragon/gomonkey"
+	getty "github.com/apache/dubbo-getty"
+	"github.com/seata/seata-go/pkg/protocol/message"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestGettyRemotingClient_SendSyncRequest unit test for SendSyncRequest function
+func TestGettyRemotingClient_SendSyncRequest(t *testing.T) {
+	respMsg := message.GlobalBeginResponse{
+		AbstractTransactionResponse: message.AbstractTransactionResponse{
+			AbstractResultMessage: message.AbstractResultMessage{
+				ResultCode: message.ResultCodeSuccess,
+			},
+		},
+	}
+	mock := gomonkey.ApplyMethod(reflect.TypeOf(GetGettyRemotingInstance()), "SendSync",
+		func(_ *GettyRemoting, msg message.RpcMessage, s getty.Session, callback callbackMethod) (interface{},
+			error) {
+			return respMsg, nil
+		})
+	resp, err := GetGettyRemotingClient().SendSyncRequest("message")
+	assert.Empty(t, err)
+	assert.Equal(t, respMsg, resp.(message.GlobalBeginResponse))
+	mock.Reset()
```

Comment:
mock.Reset() Function does not seem to be necessary here

Reply:
> mock.Reset() Function does not seem to be necessary here

Deleted